api/v1alpha1/etcdcluster_types.go

```diff
@@ -200,6 +203,41 @@ type StorageSpec struct {
 	VolumeClaimTemplate EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
 }
 
+// SecuritySpec defines security settings for etcd.
+// +k8s:openapi-gen=true
+type SecuritySpec struct {
+	// +optional
+	Peer *PeerSpec `json:"peer,omitempty"`
+	// +optional
+	ClientServer *ClientServerSpec `json:"clientServer,omitempty"`
+}
+
+type PeerSpec struct {
+	// +optional
+	Ca SecretSpec `json:"ca,omitempty"`
+	// +optional
+	Cert SecretSpec `json:"cert,omitempty"`
+}
+
+type ClientServerSpec struct {
+	// +optional
+	Ca SecretSpec `json:"ca,omitempty"`
+	// +optional
+	ServerCert SecretSpec `json:"serverCert,omitempty"`
+	// +optional
+	RootClientCert SecretSpec `json:"rootClientCert,omitempty"`
```

Comment:
I think it would be better to adhere to etcd semantics (see flag names in the security section of the [configuration guide](https://etcd.io/docs/v3.5/op-guide/configuration/)). This is covered by the flag `--trusted-ca-file`. I suggest using a field name like `TrustedCA`.

Reply:
Resolved in favor of the comment for lines 223-224.